### ISSUE TITLE
Feature Request: Implementation of getTerminatorSeq() Function

### ISSUE THREAD

**Issue by JavierMenRev** [Bioconductor/GenomicFeatures#66]
Currently, the package includes the getPromoterSeq() function, which is very useful for retrieving promoter sequences. But, there is currently no equivalent function for retrieving terminator sequences.

**Comment by hpages** [comment]
Can you be more specific about what `getTerminatorSeq()` should do, with a concrete example? Thanks

**Comment by JavierMenRev** [comment]
Sure, getPromoterSeq() currently extracts sequences for the genes or transcripts specified in the query (a [GRanges] or [GRangesList] object) from a [BSgenome] object or a [FaFile]. I was thinking of the function having the same functionality but pulling the terminators, i.e., using the transcription end site instead of the start site of each transcript/gene.

An example would be:

```
fa <- open(FaFile(...)) 
tx <- makeTxDbFromGFF(..)
genes_gr <- genes(tx)
terminators <- getTerminatorSeq(genes_gr, fa, upstream=100, downstream=100)
```

**Comment by hpages** [comment]
Thanks.

What would be good default values for the `upstream` and `downstream` arguments? For `getPromoterSeq()` we use  `upstream=2000` and `downstream=200`. Should we use the same for `getTerminatorSeq()`? I'm not familiar with the biology around terminator regions but that sounds excessive maybe?

**Comment by hpages** [comment]
Done in **GenomicFeatures** 1.55.2 (see commit 4e7cf9891d4224160d5f00dce120a4b0c3b831fb). It might take one day or two before this becomes available in BioC devel via `BiocManager::install()`.

Cheers,
H.

**Comment by JavierMenRev** [comment]
Thanks, @hpages! Could I also have the command to install that specific version? I've tried `devtools::install_github("Bioconductor/GenomicFeatures@v1.55.2")` , but it looks like devtools looks for a tar file by default which seems not to be present here?

**Comment by hpages** [comment]
`getTerminatorSeq()` was added to **GenomicFeatures** 1.55.2, which made it to Biocondcutor devel (BioC 3.19).

IMPORTANT: It only works with -- and is only available in -- Bioconductor 3.19. So, in order to get this new feature, you'll need to upgrade your installation to BioC 3.19 which requires R 4.4.

Once you have R 4.4, install **BiocManager** from CRAN. Then just do `BiocManager::install("GenomicFeatures")`. This will get you the latest **GenomicFeatures** (which is 1.55.3 as of today). 

Installing a specific version of a devel package is not a good idea. Always use `BiocManager::install()` to get the latest version of a package for _your_ version of Bioconductor. Never install a Bioconductor package directly from GitHub. 

Best,
H.
